app/src/main/java/ch/hikemate/app/ui/map/RunHikeScreen.kt

```diff
@@ -184,7 +184,11 @@ private fun runHikeMap(hike: DetailedHike): MapView {
       factory = { mapView },
       modifier =
           Modifier.fillMaxWidth()
-              .padding(bottom = 300.dp) // Reserve space for the scaffold at the bottom
+              .padding(
+                  bottom =
+                      300.dp -
+                          20.dp) // Reserve space for the scaffold at the bottom, -20.dp to avoid
+              // the map being to small under the bottomSheet
```

Comment:
Nitpick: maybe this is a good opportunity to refactor the run hike screen and put those values in constants in the `RunHikeScreen` object. The constants should not be shared between screens, as we might want to make the bottom sheet slightly higher on details screen than on map screen for example.